test/core.jl

```diff
@@ -3106,3 +3106,8 @@ end
 const DATE12003 = DateTime(1917,1,1)
 failure12003(dt=DATE12003) = Dates.year(dt)
 @test isa(failure12003(), Integer)
+
+# Test error checking in bitstype
+@test_throws ErrorException bitstype 0 SPJ001
```

Comment:
I tried to push this a few days ago with the changes @tkelman wanted, I'm not sure what happened, I pushed it again now.
In the case of a `@test_throws`, do the names even matter?  (does the name get added to the symbol table before the exception gets thrown?)
Thanks for the reviewing!